CMakeLists.txt

```diff
@@ -246,6 +246,9 @@ if (WITH_DISNEY_MATERIAL)
     endif ()
 endif ()
 
+if (WITH_STUDIO)
+    find_package (OpenColorIO REQUIRED)
```

Comment:
I'm taking care of this, as well as the Windows config. I'll make a PR to your repo.